pkg/apis/extensions/v1beta1/types.go

```diff
@@ -1036,3 +1037,58 @@ type PodSecurityPolicyList struct {
 	// Items is a list of schema objects.
 	Items []PodSecurityPolicy `json:"items"`
 }
+
+// Template contains the inputs needed to produce a Config.
```

Comment:
are we putting this in extensions, or are you planning to move it? if we know it's going to move and you're about to check this in, suggest a comment saying so (it will show up in the swagger site through docs/api-reference/extensions/v1beta1/definitions.html).